app/test_main.py

```diff
@@ -1,3 +1,47 @@
+import pytest
+from typing import List
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    # Basic test cases
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ]
+    )
+    def test_basic_cases(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: List[int]
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == expected
+
+
+class TestEdgeCases:
```

Comment:
no need to create an extra class for this - you can include this method in above class